src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -468,6 +468,22 @@ public static File createTempFile(String name, String extension) {
         }
     }
 
+
+    /**
+     * @param extension a file extension, may include 0 or more leading dots which will be replaced with a single dot
+     * @return replace the final extension on a path with the given extension
+     */
+    public static String replaceExtension(String path, String extension){
+        Utils.nonNull(path);
+        Utils.nonNull(extension);
+        final String extensionNoLeadingDot = StringUtils.stripStart(extension, ".");
+        return FilenameUtils.removeExtension(path) + '.' + extensionNoLeadingDot;
+    }
+
+    public static File replaceExtension(File file, String extension){
+        return new File(replaceExtension(file.toString(), extension));
```

Comment:
don't use `toString()` for any logic - I think you want `getAbsolutePath()` here